components/eam/src/physics/rrtmgp/radiation.F90

```diff
@@ -1656,25 +1606,16 @@ subroutine radiation_tend(state,   ptend,    pbuf,          cam_out, cam_in,  &
                      call get_curr_date(year,month,day,tod)
                      !populate with corresponding pbuf
                      !variables
-                     call pbuf_get_field(pbuf,aer_tau_bnd_lw_mon_1_idx, aer_tau_bnd_lw_mon_1)
-                     call pbuf_get_field(pbuf,aer_tau_bnd_lw_mon_2_idx, aer_tau_bnd_lw_mon_2)
-                     if (current_month .ne. month) then
-                        aer_tau_bnd_lw_mon_1 = aer_tau_bnd_lw_mon_2
-                        if (month==12) then
-                           next_month = 1
-                        else
-                           next_month = month + 1
-                        end if
-                        allocate(aerosol_optical_property_lw(pcols,pver,nlwbands,begchunk:endchunk))
-                        call get_aerosol_optical_property_from_file(next_month,'AER_TAU_LW','lwband',nlwbands,aerosol_optical_property_lw)
-                        aer_tau_bnd_lw_mon_2 = aerosol_optical_property_lw(:,:,:,state%lchnk)
-                        deallocate(aerosol_optical_property_lw)
-                        current_month = month
+                     call pbuf_get_field(pbuf,aer_tau_bnd_lw_mon_idx, aer_tau_bnd_lw_mon)
+                     if (month==12) then
```

Comment:
I think you can move both instances of this `if` statement outside of the `if do_SPA_optics` call and clean up the code.  That being said, not really important.

Reply:
But then it will get called regardless of whether it will be used or not?

Reply:
oh wait, yes, you are correct.  I thought that this call was in both cases of an if statement, but now I see there are two separate if statements.  Ignore my suggestion :-)